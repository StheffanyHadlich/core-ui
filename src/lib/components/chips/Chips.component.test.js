import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/chips/__snapshots__"
});
