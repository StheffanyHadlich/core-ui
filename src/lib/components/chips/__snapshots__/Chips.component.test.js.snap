// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Chips Default 1`] = `
<div
  className="storybook-chips sc-ifAKCX krxbjr"
>
  <h3
    className="sc-EHOje ccjuFF"
  >
    Basic Chip
  </h3>
  <div
    className="sc-chips sc-bdVaJa cRjkCS"
  >
    <span
      className="sc-bxivhb jtBzxc"
    >
      Basic Chip
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa kviEkn"
  >
    <span
      className="sc-bxivhb jtBzxc"
    >
      Basic Chip
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa jFdlss"
  >
    <span
      className="sc-bxivhb jtBzxc"
    >
      Basic Chip
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa UtTdS"
  >
    <span
      className="sc-bxivhb jtBzxc"
    >
      Basic Chip
    </span>
  </div>
  <h3
    className="sc-EHOje ccjuFF"
  >
    Clickable Chip
  </h3>
  <div
    className="sc-chips sc-bdVaJa hryNys"
    onClick={[Function]}
  >
    <span
      className="sc-bwzfXH btFtVa"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Clickable Chip
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa djSmWm"
    onClick={[Function]}
  >
    <span
      className="sc-bwzfXH ldtRcB"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Clickable Chip
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa wpkKz"
    onClick={[Function]}
  >
    <span
      className="sc-bwzfXH rwODA"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Clickable Chip
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa foufuM"
    onClick={[Function]}
  >
    <span
      className="sc-bwzfXH ldXkiL"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Clickable Chip
    </span>
  </div>
  <h3
    className="sc-EHOje ccjuFF"
  >
    Deletable Chip
  </h3>
  <div
    className="sc-chips sc-bdVaJa cRjkCS"
  >
    <span
      className="sc-bwzfXH btFtVa"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Deletable
      <span
        className="sc-htpNat dGAvWQ"
        onClick={[Function]}
      >
        <i
          className="fas fa-times"
        />
      </span>
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa kviEkn"
  >
    <span
      className="sc-bwzfXH ldtRcB"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Deletable
      <span
        className="sc-htpNat kGqNwD"
        onClick={[Function]}
      >
        <i
          className="fas fa-times"
        />
      </span>
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa jFdlss"
  >
    <span
      className="sc-bwzfXH rwODA"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Deletable
      <span
        className="sc-htpNat fwXmyb"
        onClick={[Function]}
      >
        <i
          className="fas fa-times"
        />
      </span>
    </span>
  </div>
  <div
    className="sc-chips sc-bdVaJa UtTdS"
  >
    <span
      className="sc-bwzfXH ldXkiL"
    >
      <i
        className="fas fa-star"
      />
    </span>
    <span
      className="sc-bxivhb jtBzxc"
    >
      Deletable
      <span
        className="sc-htpNat MqEmn"
        onClick={[Function]}
      >
        <i
          className="fas fa-times"
        />
      </span>
    </span>
  </div>
</div>
`;
